media/pages/sk/podmienky-pouzivania/page.html

```diff
@@ -63,7 +64,7 @@ <h3 id="definicie-2">
 
 <p id="definicie-2-1-d">
 <span>d)</span>
-Prevádzkovateľ je prevádzkovateľ portálu www.chcemvediet.sk, ktorým je
+Prevádzkovateľ je prevádzkovateľ portálu <a href="https://chcemvediet.sk/">www.chcemvediet.sk</a>, ktorým je
 Chcemvediet.sk, občianske združenie so sídlom a adresou pre doručovanie Karpatské námestie 7770/10A,
```

Comment:
Kde sa "Chcemvediet.sk" spomina ako "občianske združenie", pls daj linku na register OZ: https://ives.minv.sk/rmno/detail?id=vigh8wDZOsCbVmMMfDuyRPgsNaInnXB5VbqYmi8F5rwkJ4INSjJnIoxNaatBM7wX  (pls over, ci to je permalinka)

Reply:
Ak to nie je permalink, ale treba hackovat ich search formular, tak kaslat na to. Nechajme to bez linky.

Reply:
Nie je to permalink :ok: